packages/ruby/lib/readme/har/response_serializer.rb

```diff
@@ -69,6 +71,10 @@ def response_body
           @response.body.each.reduce(:+)
         end
       end
+
+      def content_type_is_json?
+        JSON_MIME_TYPES.include? @response.content_type
+      end
```

Comment:
How would you feel about moving this to something like `HttpRequest#has_json_body?` It would be easier to unit-test over there and it fits with the generic request responsibility of the object?

Reply:
That sounds good 👍

Reply:
Oh huh. This doesn't fit with the `HttpRequest` object. `@response` here is a `Rack::Response`. Does it make sense to wrap that in a new class to add this `has_json_body?` functionality and test it there?

Reply:
Looking at this again I think that it makes sense to just leave everything here. It doesn't seem like we gain much by wrapping the response.